runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -21,50 +21,82 @@ import scala.concurrent.duration.{ Duration, _ }
 
 object GrpcClientSettings {
 
-  /** Scala API */
-  def apply(host: String, port: Int): GrpcClientSettings =
-    // hardcoded doesn't use the resolve timeout
-    new GrpcClientSettings(host, hardcodedServiceDiscovery(host, port), port, 1.second)
+  /**
+   * Scala API
+   *
+   * Create a client hat uses a static host and port. Default configuration
+   * is loaded from reference.conf
+   */
+  def apply(host: String, port: Int)(implicit actorSystem: ActorSystem): GrpcClientSettings = {
+    // default is hardcoded
+    val defaultServiceConfig = actorSystem.settings.config.getConfig("akka.grpc.client").getConfig("\"*\"")
+      .withValue("service-name", ConfigValueFactory.fromAnyRef(host))
+      .withValue("port", ConfigValueFactory.fromAnyRef(port))
```

Comment:
Shouldn't we look for that akka.grpc.client."host" first and only use '*' if that doesn't exist?

Reply:
We could. I don't know if it is confusing to treat a `host` as a `clientName`. Atm we only look for a specific config section for the constructor that takes in a `clientName`

Reply:
Ok, let's keep it like that then.

Reply:
Keeping is fine